utils/hare-shutdown

```diff
@@ -40,7 +41,14 @@ shutdown_sequence = ('s3service', 'ios', 'confd', 'hax', 'consul')
 
 
 def _setup_logging():
-    logging.basicConfig(level=logging.INFO, format='%(message)s')
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-v", "--verbose", help="Increase output verbosity",
+                        action="store_true")
+    args = parser.parse_args()
+    if args.verbose:
+        logging.basicConfig(level=logging.DEBUG, format='%(message)s')
```

Comment:
Should the if/else statements switched? `logging.INFO` if args.verbose is true and `logging.DEBUG` in case if its false.

Reply:
DEBUG level is more verbose and it automatically includes INFO as well. So I guess there is no mistake.

Reply:
Okay.